bin/get_data.js

```diff
@@ -14,10 +14,24 @@ var workerPaths = eval(header + body + footer);
 // End of redirector.js import
 
 const tomlConfigFile = fs.readFileSync('./wrangler.toml');
-const tomlObj = TOML.parse(tomlConfigFile);
+var tomlObj = TOML.parse(tomlConfigFile);
```

Comment:
made this change, since we're modify tomlObj it felt better for it to be a var instead of a const.

Reply:
Using `let` is generally preferred over `var` nowadays, but for the purpose of this file I think either are perfectly functional.